container/common/container_hints.go

```diff
@@ -25,7 +25,7 @@ import (
 	"os"
 )
 
-var argContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
+var ArgContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
```

Comment:
What if we keep this private, and rename `GetContainerHintsFromFile` to `GetContainerHints` with no arguments (use the flag value)?